### PR TITLE
move wiki page installation to docs

### PR DESCRIPTION
**Pull request by ferrine**
Addressing #5603

### CODE REVIEW

File: docs/source/installation.md [hunk 1]
```diff
@@ -3,11 +3,102 @@
...
-## Linux
-[Linux installation guide](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Linux))
+## Linux and Mac OS
+We recommend using [Anaconda](https://www.anaconda.com/) (or [Miniforge](https://github.com/conda-forge/miniforge)) to install Python on MacOS, which allows for packages to be installed using its `conda` utility. We recommend installing PyMC into a clean `conda` environment, which here is called `pymc_env` but can be changed to anything else:
+
+```console
+conda create -n pymc_env pymc python pip
+```
+
+YOu can also use pip. Activate the environment that you wish to install into, and install as follows:
```
Comment (twiecki) [inline docs/source/installation.md]:
```suggestion
You can also use pip. Activate the environment that you wish to install into, and install as follows:
```

**Comment by OriolAbril** [comment]
Just noting that this won't show in 4.0.0 docs, if we have a 4.0.1 milestone we should probably add it there. From now on, the exact same commit used for the release will be used to build its docs.

**Comment by twiecki** [comment]
@OriolAbril Yeah, we decided to punt this to `4.0.1` because we are still fine-tuning installation instructions and the wiki is a quicker way to do so.

**Comment by twiecki** [comment]
We can probably simplify this a lot now and don't need OS-specific instructions anymore. Really the only thing required now is `conda install -c conda-forge pymc`.

**Pull request closed without merging**